extension/chrome/settings/index.ts

```diff
@@ -380,6 +384,7 @@ View.run(class SettingsView extends View {
     } catch (e) {
       if (ApiErr.isAuthPopupNeeded(e) || ApiErr.isAuthErr(e)) {
         $('#status-row #status_google').text(`g:?:auth`).addClass('bad');
+        console.log('reconect confirm')
         if (await Ui.modal.confirm(`FlowCrypt must be re-connected to your Google account.`)) {
```

Comment:
@tomholub here, how do I check that `'/chrome/settings/modules/auth_denied.htm'` modal is rendered not to render this confirm on top?

Reply:
I think the above modal solution would affect this as well, so you'd no longer need to check, since no redirect took place. Feels cleaner overall. Let me know what you think.